assets/js/googlesitekit/datastore/site/reset.test.js

```diff
@@ -85,7 +85,7 @@ describe( 'core/site reset', () => {
 
 				// After a successful reset, `connection` should be `undefined` again.
 				const connection = await registry.select( STORE_NAME ).getConnection();
-				expect( connection ).toEqual( undefined );
+				expect( connection ).toEqual( { connected: true, resettable: true } );
```

Comment:
The name of this test should also be updated to be more accurate as well.
How about: it "resets connection on server only" ? I think this would be a bit more clear as to what is going on in combination with the updated comment.
Unfortunately GitHub won't let me suggest this change because it's out of range of the current changes.

Reply:
Sure. Sounds good.